_data/calendar.yaml

```diff
@@ -4,78 +4,20 @@
 # (which will probably be a cancellation).
 # Whichever day is marked as primary will be the day that the sign-in link
 # is active for.
-- date: 2017-09-05
+
+- date: 2018-01-23
   week: 0
-  lecture: 0
-  lab: 0
+  id: 0
 
 - date: 2017-09-07
+  week: 1
+  id: b1
+  track: Basic
 
-- date: 2017-09-12
+- date: 2017-09-09
```

Comment:
Dates here is wrong

Reply:
these are all placeholder dates to test the new config file layout